infra/terraform/test-org/ci-project/sa.tf

```diff
@@ -38,5 +38,9 @@ module "oidc" {
       sa_name   = module.service_accounts.service_account.name
       attribute = "attribute.repository/GoogleCloudPlatform/cloud-foundation-toolkit"
     }
+    cft-github-actions-tgc = {
+      sa_name   = module.service_accounts.service_account.name
```

Comment:
This SA only has the permissions defined in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/8d731102d9becaaa245ea2587c4ea65b9cdbce40/infra/terraform/test-org/ci-project/sa.tf#L23-L26. It might be better to create a different SA with necessary roles.